pallets/allocations/src/weights.rs

```diff
@@ -67,13 +67,13 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: System Events (r:1 w:1)
 	/// The range of component `b` is `[1, 500]`.
 	fn allocate(b: u32) -> Weight {
-		(0 as Weight)
-			// Standard Error: 4_000
-			.saturating_add((7_946_000 as Weight).saturating_mul(b as Weight))
-			.saturating_add(T::DbWeight::get().reads(8 as Weight))
-			.saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(b as Weight)))
-			.saturating_add(T::DbWeight::get().writes(6 as Weight))
-			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(b as Weight)))
+		Weight::from_ref_time(0 as u64)
```

Comment:
You can remove all the `as u64` for integral literals in this file as suggested by clippy as well.

Reply:
Actually I remembered this is a generated code and we better not touch it.

Reply:
Actually never mind this. I re-benchmarked the allocations and the generated code would look like that.